server/src/main/java/io/druid/segment/realtime/firehose/EventReceiverFirehoseFactory.java

```diff
@@ -137,6 +139,7 @@ public int getBufferSize()
 
     private volatile InputRow nextRow = null;
     private volatile boolean closed = false;
+    private AtomicLong bytesReceived = new AtomicLong(0);
```

Comment:
its AtomicLong, I dont think we need it to be volatile, will make it final though.